src/api/app/assets/javascripts/webui/application/datatables.js

```diff
@@ -0,0 +1,59 @@
+//= require datatables/jquery.dataTables
+
+// optional change '//' --> '//=' to enable
+
+// require datatables/extensions/AutoFill/dataTables.autoFill
+// require datatables/extensions/Buttons/dataTables.buttons
+// require datatables/extensions/Buttons/buttons.html5
+// require datatables/extensions/Buttons/buttons.print
+// require datatables/extensions/Buttons/buttons.colVis
+// require datatables/extensions/Buttons/buttons.flash
+// require datatables/extensions/ColReorder/dataTables.colReorder
+// require datatables/extensions/FixedColumns/dataTables.fixedColumns
+// require datatables/extensions/FixedHeader/dataTables.fixedHeader
+// require datatables/extensions/KeyTable/dataTables.keyTable
+// require datatables/extensions/Responsive/dataTables.responsive
+// require datatables/extensions/RowGroup/dataTables.rowGroup
+// require datatables/extensions/RowReorder/dataTables.rowReorder
+// require datatables/extensions/Scroller/dataTables.scroller
+// require datatables/extensions/Select/dataTables.select
+
+//Global setting and initializer
+
+$.extend( $.fn.dataTable.defaults, {
+  responsive: true,
+  pagingType: 'full',
+  //dom:
```

Comment:
Please remove this boilerplate. If I want to know what this does I have to look it up anyway.

Reply:
![](https://media.giphy.com/media/1fbxZWBmBzN6g/giphy.gif)

Reply:
![source](https://user-images.githubusercontent.com/514785/43162423-3f9d9b46-8f8b-11e8-9436-7aa6c6c69cd6.gif)